recipes/qt_binder/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.2" %}
+# TODO: fix version string
```

Comment:
I think so.  I just removed the outdated TODO comment you highlighted.

The only other issue is that this currently requires the more recent libpng from defaults, but that can be fixed later in the feedstock once conda-forge either has its own Qt or a libpng compatible with the Qt from defaults.